.github/workflows/codeql-codescan.yml

```
@@ -0,0 +1,44 @@
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
+    steps:
+       - name: Checkout repository
+         uses: actions/checkout@v4
+
+       - name: Initialize CodeQL
+         uses: github/codeql-action/init@v3
+         with:
+          languages: ${{ matrix.language }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 11 but found 10

(indentation)